### PR TITLE
Fixes Some Popups

### PR DESCRIPTION
**Pull request by wout-codes**
Added domains: `1080phds.com,123-watch.com,123moviesco.site,1kmovies.org,afilmyhouse.blogspot.com,antsmovies.com,europix.us,fmoviess.site,maamusiq.com,movi.pk,thesouthflix.com,timesofurdu.website,tnmoviez.net,yify.stream,5xmovies.icu,akmm.linksme.xyz,moviesmon.cyou,filmy4wap1.online,linkshere.work,123moviesfx.live,s2dfree.*,marvel.to` ==> `break;case`

`streamtape.com` ==> `"prevent-window-open"`
`ovagames.com` ==> `||zxclan.com^$third-party` and a big scriptlet

### CODE REVIEW

**Comment by wout-codes** [comment]
I am already tempted to make this `$$script[wildcard="*function*break;case*"][min-length="30000"][max-length="85000"]` and `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\.|break;case \d{1}/")`, generic rules. There are a lot of domains already and sites uses this code most of the time to generate popups. @Alex-302 @AdamWr @zloyden What do you think?

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -159,9 +159,9 @@ imgdawgknuttz.com,getlec.blogspot.com#%#//scriptlet("abort-on-property-read", "z
...
...nmovies.to,lefturl.com,gamepciso.com,100count.net,gocoo.co,hdss.video,playerbiasa.xyz,akwams.*,freelitecoin.vip,freebitcoin.vip,bigb0ss.net,racaty.net,videobin.co,bdupload.asia,indishare.org,gdtot.com,livestreaming24.eu,ilinks.in,hdmovieplus.site,egy.best,egybest.*,upvid.net,filmvilag.me,bitfly.io,revivelink.com,owllink.net,goo-2o.com,akwam.*,multicanais.com,upstream.to,streamingcommunity.to,easyload.io,clipconverter.cc,readm.org,notube.net,gogo-stream.com,animesonline.cz,doctor-groups.com,kissanime.nz,gowatchseries.ac,aatfal.com,ouo.io,budcenter.site,ustreamix.to,ffmovies.co,animekisa.tv,shrinkbtc.cc,wlnk.ec,zone-annuaire.best,repackov.com,dood.*,youdbox.com,databasegdriveplayer.xyz,putlocker.style,watchmovies7.com.pk,manhwa18.com,swatchseries.to,mangatx.com,ouofly.com,playtube.ws,iir.ai,bolssc.com,linkjust1.com,fx4vip.com,mangazuki.online,elsfile.org,clicknupload.co,cima4u.io,mcloud.to,fmovies.to,aflamgulf.com,gateaflam.com,exee.io,anonymous-links.com,mediumarticles.com,pulsemens.com,aparat.cam,doramasmp4.com,jkanime.net,pcgamez-download.com,flvto.biz,movs4u.live#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\.|break;case \d{1}/")
```
Comment (ghajini) [inline EnglishFilter/sections/general_extensions.txt:164]: @wout-codes 
typo 
080phds.com > 1080phds.com

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:164]: Thanks.

**Comment by wout-codes** [comment]
@zloyden Can you review this please?

**Comment by zloyden** [comment]
@wout-codes I'll review it later, there are many changes.

File: EnglishFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -2321,7 +2323,8 @@ pogo.com#%#AG_defineProperty('pogo.intermission.staticAdIntermissionPeriod', {va
 ! https://github.com/AdguardTeam/AdguardFilters/issues/28680
 ! https://github.com/AdguardTeam/AdguardFilters/issues/28441
 ! https://github.com/AdguardTeam/AdguardFilters/issues/27743
-oladblock.me,oladblock.xyz,oladblock.services,oload.space,oload.live,oload.club,openload.pw,oload.fun#%#window.open = function() {}; document.cookie = "popcashpuCap=1"; Object.defineProperty(window,'turnoff',{value:true}); Object.defineProperty(window, 'preserve', { get: function() { return true; } });
+ovagames.com - popups
+ovagames.com,oladblock.me,oladblock.xyz,oladblock.services,oload.space,oload.live,oload.club,openload.pw,oload.fun#%#window.open = function() {}; document.cookie = "popcashpuCap=1"; Object.defineProperty(window,'turnoff',{value:true}); Object.defineProperty(window, 'preserve', { get: function() { return true; } });
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:2327]: `ovagames.com - popups` without `!`.
What's the point of adding the domain here `#%#window.open = function() {}; document.cookie = "popcashpuCap=1"; Object.defineProperty(window,'turnoff',{value:true}); Object.defineProperty(window, 'preserve', { get: function() { return true; } });`?

It seems only `window.open = function() {};` is useful here, the others are not.
So, the popup is fixed by `||zxclan.com^$third-party`, but you can also add `ovagames.com#%#//scriptlet("set-constant", "window.open", "noopFunc")`.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**